src/java/org/apache/cassandra/db/rows/EncodingStats.java

```diff
@@ -107,6 +108,29 @@ public EncodingStats mergeWith(EncodingStats that)
         return new EncodingStats(minTimestamp, minDelTime, minTTL);
     }
 
+    /**
+     * Merge one or more EncodingStats, that are lazily materialized from some list of arbitrary type by the provided function
+     */
+    public static <V, F extends Function<V, EncodingStats>> EncodingStats merge(List<V> values, F function)
+    {
+        if (values.size() == 1)
+            return function.apply(values.get(0));
+
+        Collector collector = new Collector();
+        for (int i=0, isize=values.size(); i<isize; i++)
```

Comment:
Nit:
```suggestion
        for (int i = 0, isize = values.size(); i < isize; i++)
```
Also, perhaps `size` or `iSize` instead of `isize`?

Reply:
Ouch. I'll fix the typos.